backend/app/views/spree/admin/orders/_risk_analysis.html.erb

```diff
@@ -12,7 +12,7 @@
         </strong></td>
           <td class="align-center">
             <span class="<%= @order.payments.failed.count > 0 ? 'state void' : 'state complete' %>">
-              <%= link_to "#{Spree.t 'payments_count', count: @order.payments.failed.count, default: pluralize(@order.payments.failed.count, Spree.t(:payment))}", spree.admin_order_payments_path(@order) %>
+              <%= link_to "#{@order.payments.failed.count} #{Spree::Payment.model_name.human(count: @order.payments.failed.count)}", spree.admin_order_payments_path(@order) %>
```

Comment:
I'm not shure about this one. I don't like to set grammatical order in views like this. Imagine a language where numbers always follow the word. This way these languages are not able to do so.